lib/build.js

```diff
@@ -443,13 +443,14 @@ function createWebpackMiddleware () {
   const clientCompiler = webpack(clientConfig)
   this.clientCompiler = clientCompiler
   // Add the middleware to the instance context
-  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, {
+  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, Object.assign(clientConfig.devServer||{},{
```

Comment:
https://webpack.js.org/configuration/dev-server/#devserver
Seems only a subset of devSever options are directly usable for devMiddleware. That Naming  may make some confuctions about available options :)